goldmane/pkg/server/flow_service.go

```diff
@@ -48,7 +48,7 @@ func (s *FlowsServer) List(req *proto.FlowListRequest, server proto.Flows_ListSe
 	}
 
 	// Send flows.
-	for _, flow := range flows {
+	for flow := range flows {
```

Comment:
Another naming thing, but I think we need to rename all of these from "flow" and "flows" to "messages" and "message" or something like that - we have a new type of non-flow message that can be sent (metadata) so we should make sure naming reflects that. 

Same for the filter hints.